migrations/sql/V2020.02.12.1100__AddModelMeta.sql

```diff
@@ -0,0 +1,72 @@
+--- What this Migration is about
+--- comment on model.is_current
+--- &
+--- creating three new table
+--- 1. model_version_meta
+--- 2. model_version_vaccine
+--- 3. model_version_country
+ALTER table model_version
+	DROP COLUMN note,
+	DROP COLUMN fingerprint;
+
+COMMENT ON COLUMN model.is_current IS
+  'TRUE if the model is currently running for VIMC.';
+
+CREATE TABLE model_version_meta (
+  id SERIAL,
+  model_version INTEGER NOT NULL,
+  is_dynamic BOOLEAN NOT NULL, 
```

Comment:
is_dynamic (and code) were added to the table model_version earlier - do we want to do something about that too?